packages/databricks-vscode/src/configuration/ConnectionManager.ts

```diff
@@ -60,10 +51,74 @@ export class ConnectionManager {
 
     public metadataServiceUrl?: string;
 
+    private async setSyncDestinationMapper() {
+        const workspacePath = await this.configModel.get("workspaceFsPath");
+        const remoteUri = workspacePath
```

Comment:
I noticed that in the `login` method we are also validating the remote uri. Does it make sense to do it here too, since this logic can be triggered outside of the login flow?

Reply:
Yep. Added.